critiquebrainz/db/review.py

```diff
@@ -0,0 +1,587 @@
+import sqlalchemy
+from critiquebrainz import db
+from critiquebrainz.db import exceptions as db_exceptions
+from flask import abort
+from brainzutils import cache
+from werkzeug.exceptions import BadRequest
+from random import shuffle
+from flask_babel import lazy_gettext
+import critiquebrainz.db.revision as db_revision
+from critiquebrainz.db.user import User
+import critiquebrainz.db.users as db_users
+import uuid
+from datetime import datetime, timedelta
+import pycountry
+
+REVIEW_CACHE_NAMESPACE = "Review"
+DEFAULT_LICENSE_ID = "CC BY-SA 3.0"
+DEFAULT_LANG = "en"
+ENTITY_TYPES = [
+    "event",
+    "place",
+    "release_group",
+]
+
+supported_languages = []
+for lang in list(pycountry.languages):
+    if 'iso639_1_code' in dir(lang):
+        supported_languages.append(lang.iso639_1_code)
+
+def get_or_404(review_id):
```

Comment:
We probably shouldn't have this function here. As I described in another comment, this makes `db` package closely tied to Flask itself. This function can be moved into packages with views if it's used more than once.